test/spec/hierarchical-facets/breadcrumb.js

```diff
@@ -68,10 +54,5 @@ test('hierarchical facets: using getHierarchicalFacetBreadcrumb on an undefined
   var client = algoliasearch(appId, apiKey);
   var helper = algoliasearchHelper(client, indexName, {});
 
-  t.throws(
-    helper.getHierarchicalFacetBreadcrumb.bind('categories'),
-    'we get an exception if the facet is not defined'
-  );
-
-  t.end();
+  expect(helper.getHierarchicalFacetBreadcrumb.bind('categories')).toThrow();
```

Comment:
Can we use an inline snapshot now that we use Jest?

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278627422.